src/datagrid/DataGridFilter.tsx

```diff
@@ -228,6 +237,14 @@ export class DataGridFilter extends React.PureComponent<DataGridFilterProps, Dat
             }
             return child;
         });
+        let alignment;
+        if (position === FilterPosition.RIGHT) {
+            alignment = "195px";
```

Comment:
Can we use rems instead ?  Verify that these filters render properly when you resize your screens.  It should always stick to the column where Filter has opened. 

Check following:

1) open filter
2) resize your screen
3) check that filter stays with the column

Reply:
done